tests/broker-engine/services-and-bulk-stmt.robot

```diff
@@ -286,67 +286,117 @@ RLCode
-    ${start}    Get Current Date
+    ${start}=    Get Current Date
 
     Start Broker
     Start Engine
 
-    ${content}    Create List    check_for_external_commands()
-    ${result}    Find In Log With Timeout    ${engineLog0}    ${start}    ${content}    60
-    Should Be True    ${result}    A message telling check_for_external_commands() should be available.
 
-    ${content}    Create List    lua: initializing the Lua virtual machine
-    ${result}    Find In Log With Timeout    ${centralLog}    ${start}    ${content}    30
-    Should Be True    ${result}    "lua logs produced"
+    ${content}=    Create List    check_for_external_commands()
+    ${result}=    Find In Log with Timeout    ${engineLog0}    ${start}    ${content}    60
+    Should Be True    ${result}    msg=A message telling check_for_external_commands() should be available.
+
+    ${content}=    Create List    lua: initializing the Lua virtual machine
+    ${result}=    Find In Log With Timeout    ${centralLog}    ${start}    ${content}    30
+    Should Be True    ${result}    msg="lua logs produced"
 
     # Define the new content to take place of the first one
-    ${new_content}    Catenate
+    ${new_content}=    Catenate
     ...    function init(params)
-    ...    broker_log:set_parameters('/tmp/titi.log', 2)
+    ...        broker_log:set_parameters('/tmp/titi.log', 2)
     ...    end
     ...
     ...    function write(d)
-    ...    broker_log:info(0, "titi")
-    ...    return true
+    ...        broker_log:info(0, "titi")
+    ...        return true
     ...    end
 
+
     # Create the second LUA script file
     Create File    /tmp/toto.lua    ${new_content}
-    ${start}    Get Current Date
+    ${start}=    Get Current Date
 
     Reload Broker
 
-    ${content}    Create List    lua: initializing the Lua virtual machine
-    ${result}    Find In Log With Timeout    ${centralLog}    ${start}    ${content}    30
-    Should Be True    ${result}    lua file not initialized
+    ${content}=    Create List    lua: initializing the Lua virtual machine
+    ${result}=    Find In Log With Timeout    ${centralLog}    ${start}    ${content}    30
+    Should Be True    ${result}    msg=lua file not initialized
+
+
+    Stop Engine
+    Kindly Stop Broker
+
+metric_mapping
+    [Documentation]    check if metric name is exist using lua conf
+    [Tags]    broker    engine    bbdo    unified_sql    metric
+    Clear Commands Status
+    Clear Retention
+
+    Remove File    /tmp/test4.log
+    Config Engine    ${1}    ${1}    ${10}
+    Config Broker    central
+    Config Broker    module
+    Broker Config Add Item    central    bbdo_version    3.0.1
+    Broker Config Add Item    module0    bbdo_version    3.0.1
+    Broker Config Log    central    lua    debug
+    Broker Config Log    module0    neb    debug
+    Config Broker Sql Output    central    unified_sql
 
+    Broker Config Add Lua Output    central    test-metric    ${SCRIPTS}test-metric.lua
```

Comment:
I want test-metric.lua to be created in this robot file. Somethink like what you did on line 328.

Reply:
done